phpBB/language/en/composer.json

```diff
@@ -22,6 +22,10 @@
 		"language-iso": "en",
 		"english-name": "British English",
 		"local-name": "British English",
-		"phpbb-version": "4.0.0-a1-dev"
+		"phpbb-version": "4.0.0-a1-dev",
+		"direction": "ltr",
+		"user-lang": "en-gb",
+		"plural-rule": "1",
+		"recaptcha-lang": "en-GB"
```

Comment:
I have no good idea yet for that case. Cause it is used within JS/Template Code only, in frontend and ACP.

Maybe simulated a 'RECAPTCHA_LANG' lang key somehow?

Reply:
Set it in `page_header()`  like `L_RECAPTCHA_LANG => $language_file_helper->get_lang_key_value('recaptcha-lang');` and change template files respectively?